src/utils/schemaValidators.js

```diff
@@ -1,10 +1,21 @@
-const OpenAPISchemaValidator = require('openapi-schema-validator').default;
-const openApi3Validator = new OpenAPISchemaValidator({
-    version: 3,
-    version3Extensions: {
-    }
-});
+
+const getOAI3Validator = (() => {
```

Comment:
why not just make it a named function?

Reply:
you're right, i had something else on mind.